src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -513,9 +513,19 @@
   - name: Create without role assignment.
     text: az ad sp create-for-rbac
   - name: Create using a custom display name.
-    text: az ad sp create-for-rbac -n "MyApp"
-  - name: Create with a Contributor role assignments on specified scopes. To retrieve current subscription ID, run `az account show --query id --output tsv`.
-    text: az ad sp create-for-rbac -n "MyApp" --role Contributor --scopes /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup1} /subscriptions/{subscriptionId}/resourceGroups/{resourceGroup2}
```

Comment:
How about simply replace `{subscriptionId}` with `00000000-0000-0000-0000-000000000000` and replace `{resourceGroup1}` with `myresourcegroup`. The example text would be:
```
text: az ad sp create-for-rbac -n "MyApp" --role Contributor --scopes /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myresourcegroup1 /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myresourcegroup2
```